### DISCUSSION TITLE
vpp version issue

### DISCUSSION THREAD

**Discussion by xcxinng** [FDio/govpp#88] [Q&A] [answered]
I used govpp on vpp:22.02.0 and got an error: stat segment version is not supported: 2 (minimal version: 1).

Is it possible to use the govpp to connect the stat segment on vpp:22.02.10? 

If no way to do that, what can I do to make it possible?

**Comment by ondrej-fabry** [comment, marked as answer]
What version of GoVPP are you using? Ensure you have the latest version. The latest version of GoVPP supports stats v2. You can update with this command:

```sh
go get -u go.fd.io/govpp@latest
```

**Reply by xcxinng** [reply]
Problem solved, thanks 🥰
